packages/next-swc/crates/next-core/src/util.rs

```diff
@@ -352,11 +366,10 @@ pub async fn load_next_js_template(
 #[turbo_tasks::function]
 pub fn virtual_next_js_template_path(
     project_path: Vc<FileSystemPath>,
-    path: String,
+    file: String,
 ) -> Vc<FileSystemPath> {
-    get_next_package(project_path)
-        .join("dist/esm".to_string())
-        .join(path)
+    debug_assert!(!file.contains('/'));
```

Comment:
is this intended assertion?

Reply:
Yes, so that we don't accidentally try to import `esm/build/templates/build/templates/pages.ts`